File: bench/src/lua/alloc.lua
local total = 0
for iter = 1, 2000 do
    local xs = {}
    for j = 1, 1000 do xs[#xs + 1] = j end
    local s = 0
    for k = 1, #xs do s = s + xs[k] end
    total = total + s
end
os.exit(total % 256)

File: bench/src/go/alloc.go
package main
import "os"
func main(){
    var total uint64
    for iter:=0; iter<2000; iter++ {
        xs := []int32{}
        for j:=int32(1); j<=1000; j++ { xs = append(xs, j) }
        var s uint64
        for _, v := range xs { s += uint64(v) }
        total += s
    }
    os.Exit(int(total % 256))
}
